frontend/components/modals/channel_form_modal.jsx

```diff
@@ -34,8 +35,15 @@ class ChannelFormModal extends React.Component {
     this.props.formAction(this.state, this.props.selectedUsers);
   }
 
+  disabledStatus(){
+    if (!this.state.name){
+      return true;
+    }
+    return this.state.private && Object.keys(this.props.selectedUsers).length == 0;
```

Comment:
this line goes over the gray line on the right. refactoring idea: extract `selectedUsers` from `props` in the line above. and/or `private` from `this.state`

Reply:
I extracted selectedUsers rather than private. If I could go back in time I would have given that column a different name because private is actually a reserved keyword so if you just call something private it can screw a lot of stuff up.